vars/testKomodo.groovy

```diff
@@ -137,7 +138,7 @@ def call(Map args = [:]) {
 
                         set -x
                         git log -n 1
-                        source ${env.CI_SOURCE_ROOT}/ci/jenkins/testkomodo.sh
+                        source ${env.CI_SOURCE_ROOT}/ci/jenkins/${params.TEST_SCRIPT}
```

Comment:
Could we make this relative to the source root, and not the jenkins folder? :thinking: